app/routes/home.js

```diff
@@ -2,8 +2,11 @@ module.exports = {
   method: 'GET',
   path: '/',
   options: {
-    handler: (request, h) => {
-      return h.view('home')
+    handler: async (request, h) => {
+      return h.view('home', {
+        totalHolds: 0,
```

Comment:
Should this be calculating the actual counts for both holds and schemes?

Currently there are 2 schemes, but the cars/bucket is showing zero in the UI?

Reply:
Yes it should.  Fixed.